core/pkg/model/kubemodel/kubemodel.go

```diff
@@ -0,0 +1,296 @@
+package kubemodel
+
+import (
+	"errors"
+	"fmt"
+	"time"
+)
+
+// @bingen:generate[stringtable]:KubeModelSet
+type KubeModelSet struct {
+	Metadata               *Metadata                         `json:"meta"`                  // @bingen:field[version=1]
```

Comment:
so should we remove the diagnostic results from each struct then ?

Reply:
That seems right to me